packages/tools/generator-pie-component/README.md

```diff
@@ -42,45 +42,17 @@ An interactive prompt should now be displayed asking for a component name.
 
 Once you have completed all the prompts, your scaffolded component will be generated! 🎉
 
-The newly generated `package.json` file will most likely contain references to the local monorepo packages. Please replace its versions with the latest published version:
-
-```sh
-cd packages/components/pie-{{COMPONENT_NAME}}
-npx npm-check-updates "@justeattakeaway/*" -u
-```
-
-_Note: If this step fails, ensure you have installed the repository dependencies with `yarn`. Otherwise, this (and any other commands) will fail._
-
-### Setting up Storybook:
-
-- Add the component to the storybook package.json dependencies:
-
-```json
-"@justeattakeaway/pie-{{COMPONENT_NAME}}": "0.0.0",
-```
-
 ### Setting up Visual Regression Testing:
 
 - Create a Percy project for the component on the Percy website.
-- Update the `test:visual` command in package.json to the following:
-
-```sh
-"test:visual": "run -T cross-env-shell PERCY_TOKEN=${PERCY_TOKEN_PIE_COMPONENT_NAME} percy exec --allowed-hostname cloudfront.net -- npx playwright test -c ./playwright-lit-visual.config.ts",
-```
-
-- Add the token environment variable to `github/workflows/ci.yml` as follows:
+- Update the component token environment variable in `github/workflows/ci.yml` as follows:
```

Comment:
I don't think this file will need manually updating, I think you've fully automated it 😄

Reply:
Updated the docs